### DISCUSSION TITLE
end of maestral log show

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [samschott/maestral#588] [Q&A] [answered]
When I type `maestral log show`, how can I go to the end?

**Comment by samschott** [comment, marked as answer]
The log is shown with a so-called "pager", a program which lets you view and move through text files without editing them. The actual pager will probably be [`less`](https://en.wikipedia.org/wiki/Less_(Unix)), if it is available on your system.

Some useful commands are: `q` to quit and `h` for help with an overview of all commands.

To answer your actual question: You can jump to the last line with either `G` or `>`.

**Reply by raffaelemancuso** [reply]
> The log is shown with a so-called "pager", a program which lets you view and move through text files without editing them. The actual pager will probably be [`less`](https://en.wikipedia.org/wiki/Less_(Unix)), if it is available on your system.
> 
> Some useful commands are: `q` to quit and `h` for help with an overview of all commands.
> 
> To answer your actual question: You can jump to the last line with either `G` or `>`.

Yeah, or maybe just "end".

Thank you Sam
